synapse/lib/stormtypes.py

```diff
@@ -4805,9 +4806,10 @@ async def _methUsersByName(self, name):
         if udef is not None:
             return User(self.runt, udef['iden'])
 
-    async def _methUsersAdd(self, name, passwd=None, email=None):
+    async def _methUsersAdd(self, name, passwd=None, email=None, iden=None):
```

Comment:
while we're here, we should probably add tostr() calls quick ( with noneok=True for the optional ones )